actors/miner/src/lib.rs

```diff
@@ -30,6 +30,8 @@ use log::{error, info, warn};
 use num_derive::FromPrimitive;
 use num_traits::{Signed, Zero};
 
+use crate::ext::market::DealSpaces;
```

Comment:
FYI In my upcoming refactor of ActivateDeals I have purged use of this type from the miner actor, but added a new type, tentatively called `DealActivationInfo` for the return from `batch_activate_deals_and_claim_allocations`. It includes CommD if requested.

We need to clarify a naming convention for the inputs vs the results of activation. For a verbose but unmistakable convention, how about DataActivationInput/Output (and similar for Sector)?